crates/openid4vci/src/jwt/mod.rs

```diff
@@ -195,7 +195,7 @@ impl ProofJwt {
                         }
                     })?;
 
-                    Ok(Some(did.to_string()))
+                    Ok(Some(did.did))
```

Comment:
This method is not really accurate anymore now. The kid refers to a key identifier, while if we return just the did, it's not an identifier to a specific key anymore

Reply:
I will rename it to `extract_did`

Reply:
👍